packages/config/config/devices/0x0344/he-ht01.json

```diff
@@ -265,21 +194,13 @@
 		},
 		{
 			"#": "16",
-			"label": "Floor Temperature Calibration",
-			"unit": "0.1 °C",
-			"valueSize": 1,
-			"minValue": -100,
-			"maxValue": 100,
-			"defaultValue": 0
+			"$import": "templates/heltun_template.json#temp_sensor_cal",
```

Comment:
I'd prefer if the template keys weren't abbreviated too much. --> `temp_sensor_calibration`